ledger/participant-integration-api/src/main/scala/db/migration/postgres/V38__Update_value_versions.scala

```diff
@@ -73,18 +71,17 @@ private[migration] final class V38__Update_value_versions extends BaseJavaMigrat
       nameValue: (String, Option[Value]),
       tableName: String,
       rowKey: String
-  ): NamedParameter =
-    nameValue match {
-      case (label, Some(value)) =>
-        NamedParameter(
-          label,
-          ValueSerializer
-            .serializeValue(
-              value.copy(stableValueVersion),
-              s"failed to serialize $label for $tableName $rowKey"))
-      case (label, None) =>
-        NamedParameter(label, EmptyArray)
-    }
+  ): NamedParameter = {
```

Comment:
Perhaps if this returned an `Option` and it was converted into a `NamedParameter` in a separate operation, the logic would be clearer.

Alternatively you could add an explicit type to `newValue`.

What do you think?

Reply:
My initial idea was to re-write the thing to make it dumber, but I'd prefer changing this as little as possible.